src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -371,8 +371,12 @@ private void addGraalStaticLibsPathToLinkProcess(ProcessBuilder linkBuilder) {
     }
 
     private void addJavaStaticLibsPathToLinkProcess(ProcessBuilder linkBuilder) throws IOException {
-        Path javaSDKPath = fileDeps.getJavaSDKPath();
-        linkBuilder.command().add(getLinkLibraryPathOption() + javaSDKPath);
+        if (useGraalVMJavaStaticLibraries()) {
```

Comment:
Is there any chance that this can be selected from within internal Configuration, so we can try to use java sdk from  a current path?

Reply:
Can you explain in more detail what you mean by this?

Reply:
With your changes, `javaSDKPath` becomes irrelevant (I guess that is purpose?), but we still have `useCustomJavaStaticLibs()` in our InternalProjectConfiguration, so maybe we still need to keep this option? @johanvos thoughts?